scripts/test-ci-all.sh

```diff
@@ -89,17 +89,27 @@ export -f cli_test_rust_tests_electrs
 
 function cli_test_rust_tests_esplora() {
   set -eo pipefail # pipefail must be set manually again
-  trap 'echo "## FAILED: ${FUNCNAME[0]}"' ERR 
+  trap 'echo "## FAILED: ${FUNCNAME[0]}"' ERR
 
   echo "## START: ${FUNCNAME[0]}"
   unshare -rn bash -c "ip link set lo up && exec unshare --user env FM_TEST_ONLY=esplora ./scripts/rust-tests.sh" 2>&1 | ts -s
   echo "## COMPLETE: ${FUNCNAME[0]}"
 }
+
+function cli_test_wasm() {
+  set -eo pipefail # pipefail must be set manually again
+  trap 'echo "## FAILED: ${FUNCNAME[0]}"' ERR
+
+  echo "## START: ${FUNCNAME[0]}"
+  unshare -rn bash -c "ip link set lo up && exec unshare --user env FM_TEST_ONLY=esplora ./scripts/wasm-tests.sh" 2>&1 | ts -s
+  echo "## COMPLETE: ${FUNCNAME[0]}"
+}
+
```

Comment:
It seems like this function isn't getting called anywhere. I don't think these tests are actually being run in CI. I think you need to add this to the `parallel` command for it to actually run.